readme.md

```diff
@@ -217,7 +217,6 @@ Tools and resources that provide functionality for presenting IIIF materials in
 
 - [Adno](https://adno.app/en/) - a web application for viewing, editing and sharing narratives and pathways on IIIF images.
 - [Annona Range Storyboard](https://ncsu-libraries.github.io/annona/range/) - [Annona](https://ncsu-libraries.github.io/annona/) toolkit which allows for the guided viewing of segments of a manifest, in addition to the [Annona Multi Storyboard Viewer](https://ncsu-libraries.github.io/annona/multistoryboard/) for guided comparison of multiple manifests.
-- [Bloom IIIF](https://samvera-labs.github.io/bloom-iiif/) - IIIF Presentation API Collection items React.js component that renders a horizontal slider
```

Comment:
Totally okay with getting rid of this. Though technically it is now the Slider component at https://samvera-labs.github.io/clover-iiif/docs/slider